docs/tutorials/multimodal/advanced_topics/tensorrt.ipynb

```diff
@@ -0,0 +1,411 @@
+    "    \"model.timm_image.checkpoint_name\": \"mobilenetv3_small_100\",\n",
+    "    \"model.hf_text.checkpoint_name\": \"google/electra-small-discriminator\",\n",
+    "    \n",
+    "}\n",
+    "predictor = MultiModalPredictor(label=label_col).fit(\n",
+    "    train_data=train_data,\n",
+    "    hyperparameters=hyperparameters,\n",
+    "    time_limit=120, # seconds\n",
+    ")\n",
+    "\n",
+    "clear_output()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae8e9de8",
+   "metadata": {},
+   "source": [
+    "Under the hood, AutoMM automatically infers the problem type (classification or regression), detects the data modalities, selects the related models from the multimodal model pools, and trains the selected models. If multiple backbones are available, AutoMM appends a late-fusion model (MLP or transformer) on top of them.\n",
+    "\n",
+    "## Prediction with default PyTorch module\n",
+    "Given a multimodal dataframe without the label column, we can predict the labels.\n",
+    "\n",
+    "Note that we would use a small sample of test data here for benchmarking. Later, we would evaluate over the whole test dataset to assess accuracy loss."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b0bc369",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "batch_size = 2\n",
+    "n_trails = 10\n",
+    "sample = test_data.head(batch_size)\n",
+    "\n",
+    "# Use first prediction for initialization (e.g., allocating memory)\n",
+    "y_pred = predictor.predict_proba(sample)\n",
+    "\n",
+    "pred_time = []\n",
+    "for _ in range(n_trails):\n",
+    "    tic = time.time()\n",
+    "    y_pred = predictor.predict_proba(sample)\n",
+    "    elapsed = time.time()-tic\n",
+    "    pred_time.append(elapsed)\n",
+    "    print(f\"elapsed (pytorch): {elapsed*1000:.1f} ms (batch_size={batch_size})\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "912bdb80",
+   "metadata": {},
+   "source": [
+    "## Prediction with TensorRT module\n",
+    "First, we need to export the module to ONNX, in order to use TensorrtExecutionProvider in onnxruntime for prediction."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0448f763",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_path = predictor.path\n",
+    "trt_predictor = MultiModalPredictor.load(path=model_path)\n",
+    "trt_predictor.optimize_for_inference()\n",
+    "\n",
+    "# Agagin, use first prediction for initialization (e.g., allocating memory)\n",
+    "y_pred_trt = trt_predictor.predict_proba(sample)\n",
+    "\n",
+    "clear_output()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0bf01e0",
+   "metadata": {},
+   "source": [
+    "Under the hood, the `optimize_for_inference()` would generate an OnnxModule that can be a drop-in replacement of torch.nn.Module, and it replaced torch.nn.Module for optimized inference with onnxruntime."
```

Comment:
Users probably don't know what is`OnnxModule ` defined by us. Better to replace it with  more general descriptions.

Reply:
replaced with general terms.